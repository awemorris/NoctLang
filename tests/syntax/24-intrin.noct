func main() {
    var i = int("100");
    print(i);

    var f = float("100.0");
    print(f);
}
