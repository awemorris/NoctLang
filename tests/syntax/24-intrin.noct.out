100
100.000000
